examples/image-streams/image-streams-centos7.json

```diff
@@ -14,7 +14,11 @@
         "dockerImageRepository": "openshift/ruby-20-centos7",
```

Comment:
Remove this and every occurence, where you specify images to be imported manually.